.github/workflows/twister_tests.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           ${{ runner.os }}-pip-${{ matrix.python-version }}
     - name: install-packages
       run: |
-        pip3 install pytest colorama pyyaml ply mock pykwalify
+        pip3 install -r scripts/requirements.txt
```

Comment:
Why can't we just do `pip3 install pytest colorama pyyaml ply mock pykwalify psutil` here?

Installing all the packages from `requirements.txt` is going to make this workflow take longer and more subject to breakages.

Reply:
also note that we have `scripts/requirements-run-test.txt`

Reply:
Adding `psutil` directly would work, but I was hoping to avoid continuing to have two different lists of Twister dependencies as it could lead to the same problem happening again in the future.

I looked at `scripts/requirements-*.txt` and it looks like the minimal set of requirements would be `requirements-base.txt` plus `requirements-build-test.txt`:

```
cat requirements-base.txt requirements-build-test.txt | grep -i "^pytest\|^colorama\|^pyyaml\|^ply\|^mock\|^pykwalify\|^psutil"
PyYAML>=5.1
pykwalify
psutil
colorama
ply>=3.10
pytest
mock>=4.0.1
```

But this still pulls in several unused packages. Maybe we should do some re-organization of these files?

Reply:
@stephanosio what do you think about installing just `requirements-base.txt` and `requirements-build-test.txt`?

Reply:
> @stephanosio what do you think about installing just `requirements-base.txt` and `requirements-build-test.txt`?

That still has a lot more than "bare minimum," but sure. We will be refactoring this workflow soon anyway so these packages are installed in the Docker image.

Reply:
I'll upload a new commit that uses just those two requirements files.